Util/Const.lua

```diff
@@ -6,6 +6,22 @@ Arenalogs.CONST = {}
 
 local f = Arenalogs.GUTIL:GetFormatter()
 
+classColors = {

```

Comment:
If you dont specify `local` for a new variable it will be **global**
Which is to be avoided as much as possible

Also you dont actually have to have a classcolor table to color text by class
you can use GUTIL's formatter for that 

In this file its already included (line 7) as **f**
so you can just
`f.class("Some Test", "PALADIN")`
and you get a string that is formatted with the PALADIN class color